code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -98,7 +98,7 @@
 		"<span class='userdanger'>You feast on [L], restoring your health!</span>")
 	if(!is_station_level(z) || client) //NPC monsters won't heal while on station
 		adjustBruteLoss(-L.maxHealth/2)
-	L.gib()
+	L.gib(no_drops=1)
```

Comment:
TRUE this.

Reply:
Fug, I've re-done this so many times that this one has slipped through.